src/formatDate.js

```diff
@@ -8,7 +8,26 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDateFormat = date.split(fromFormat[fromFormat.length - 1]);
+  const newDateFormat = [];
+
+  for (let i = 0; i < oldDateFormat.length; i++) {
+    if (toFormat.indexOf(fromFormat[i]) === -1) {
+      if (fromFormat[i].length === 2) {
+        if (+oldDateFormat[i] < 30) {
+          newDateFormat[toFormat.indexOf('YYYY')] = '20' + oldDateFormat[i];
+        } else {
+          newDateFormat[toFormat.indexOf('YYYY')] = '19' + oldDateFormat[i];
```

Comment:
The code converts two-digit years to four-digit years by prefixing '20' or '19', but it does not account for the possibility of having years outside the 20th or 21st century. This may not be an issue depending on the context of the application, but it's worth noting if historical or future dates are within the scope of the function.